src/main/java/core/basesyntax/Reader.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class Reader {
+    public int[] readFromFile(String fileName) {
+        int supply = 0;
+        int buy = 0;
+        File file = new File(fileName);
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));

```

Comment:
use try with resources
go through common mistakes list and check yourself once more